Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -598,27 +594,29 @@ def create_from_rows(self, relevant_keys, relevant_ids, selection=False):
 
     def expand_table(self, table, atrs, metas, cv):
         exp = []
-        row_vals = {'attributes' : 'x', 'class_vars' : 'y', 'metas' : 'metas'}
-        l = 1 if isinstance(table, RowInstance) else len(table)
-        ids = table.id.reshape(-1,1) if isinstance(table, RowInstance) else table.ids.reshape(-1, 1)
-        atr_vals = row_vals if isinstance(table, RowInstance) else self.atr_vals
+        n = 1 if isinstance(table, RowInstance) else len(table)
```

Comment:
`l` is not a safe name because of its potential similarity with `1` in some fonts.